samples/nrf9160/serial_lte_modem/src/main.c

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <logging/log.h>
+LOG_MODULE_REGISTER(app, CONFIG_THIN_LOG_LEVEL);
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <uart.h>
+#include <string.h>
+#include <net/socket.h>
+#include <init.h>
+#include <bsd.h>
+#include <lte_lc.h>
+#include <net/mqtt.h>
+
+#include <hal/nrf_gpio.h>
+#include <hal/nrf_power.h>
+#include <hal/nrf_regulators.h>
+
+#include "th_test.h"
+#include "th_connect.h"
+#if defined(CONFIG_LTE_LINK_CONTROL)
+#include "th_modem_service.h"
+#endif
+#if defined(CONFIG_THIN_MQTT_SERVICE)
+#include "th_mqtt_service.h"
+#endif
+#if defined(CONFIG_THIN_LWM2M_SERVICE)
+#include "th_lwm2m_service.h"
+#endif
+#if defined(CONFIG_THIN_GPS_SERVICE)
+#include "th_gps_service.h"
+#endif
+#if defined(CONFIG_THIN_SNTP_SERVICE)
+#include "th_sntp_service.h"
+#endif
+#if defined(CONFIG_THIN_TCPIP_SERVICE)
+#include "th_tcpip_service.h"
+#endif
+
+#define INVALID_DESCRIPTOR	-1
+
+#define AT_BUF_SIZE		CONFIG_INTER_CONNECT_UART_BUF_SIZE
+#define THREAD_STACK_SIZE	1048
```

Comment:
I suppose 1024 was meant here? Or is this finetuned to 1048?

Reply:
You could use `KB(x)` from `zephyr/include/sys/util.h` here, if `1048` is a mistake.
Otherwise a short comment could clarify that `1048` is correct.

Reply:
It's a typo and updated to KB(1), Thanks.